### ISSUE TITLE
GhiHorn PathAnalyzer: Cannot invoke "ghidra.program.model.pcode.HighVariable.getDataType()" because "highVariable" is null

### ISSUE THREAD

**Issue by JeroenRobbenFirmalyzer1** [CERTCC/kaiju#23]
Hi,

I'm running into a problem while using the GhiHorn path analyzer on a ARMv7-M firmware elf (with symbols). The tool emits error "Cannot invoke "ghidra.program.model.pcode.HighVariable.getDataType()" because "highVariable" is null" and doesn't return any results. In the Ghidra 'Decompile' pane the function seems to be decompiled fine and function signature is OK.

![error](https://user-images.githubusercontent.com/97623479/166930235-f27ae9f5-9bf1-4750-b3cb-57cdd7c1cdad.png)

**Steps to reproduce:**
1. Unzip the attached [sample.elf.zip](https://github.com/CERTCC/kaiju/files/8631509/sample.elf.zip)
2. Import 'sample.elf', pick Arm Cortex 32-bit little-endian as isa. 
3. Run all analyzers
4. Open the GhiHorn tab (ctrl+g). 
5. Set "2ecac" as program entrypoint ("main" function), as end address "0002ecb2"
6. Click 'Analyze'

**Environment:**
 - OS: Ubuntu 22.04, Java 15.0.2
 - z3 4.8.14.0, Ghidra 10.1.3
 - Latest Kaiju.zip from Github releases (commit 40d83ed)

Thanks!

**Comment by sei-gwassermann** [comment]
There was from what I recall some kind of problem with Ghidra's HighVariable implementation that was fixed upstream in Ghidra 10.2. Will take a look to see if I can confirm that this issue may be fixed. (Probably should have better error handling either way).

**Comment by sei-gwassermann** [comment]
Unfortunately Ghidra 10.2 did not fully resolve the issue. I did find the location where the null is apparently introduced:

https://github.com/CERTCC/kaiju/blob/042be73a14d350bcb1a13d26ed86ade507f2c2bd/src/main/java/kaiju/tools/ghihorn/hornifer/GhiHornifier.java#L1050

I'm not sure why this is happening; it queries Ghidra's implementation, so apparently we're getting a null from Ghidra.

I patched the issue to look for a null and use an unknown variable instead if it is null, so this at least gets rid of the error message. Will need to investigate what's going on and if it is impacting analysis results.
